starfish/core/spots/_detect_spots/blob.py

```diff
@@ -115,7 +115,7 @@ def image_to_spots(self, data_image: Union[np.ndarray, xr.DataArray]) -> SpotAtt
         fitted_blobs[Features.SPOT_RADIUS] = converted_radius
 
         # convert the array to int so it can be used to index
-        rounded_blobs = SpotAttributes(fitted_blobs.astype(int))
+        rounded_blobs = SpotAttributes(fitted_blobs)
```

Comment:
rename ```rounded_blobs``` -> ```spots```? since they're no longer rounded, and they're actually now spots as defined by SpotAttributes

Reply:
so would we do a search-and-replace of `blob`->`spot`?  i.e., would `fitted_blobs` be renamed to `fitted_spots`?

Reply:
i think fitted_blobs should stay fitted_blobs, as it's the output of a blob detector. my sense is no need for the search-and-replace.